luci-app-passwall/luasrc/model/cbi/passwall/client/acl_config.lua

```diff
@@ -331,8 +331,7 @@ if has_singbox or has_xray then
 	o:value("https://1.1.1.2/dns-query", "CloudFlare-Security")
 	o:value("https://8.8.4.4/dns-query", "Google 8844")
 	o:value("https://8.8.8.8/dns-query", "Google 8888")
-	o:value("https://9.9.9.9/dns-query", "Quad9-Recommended 9.9.9.9")
-	o:value("https://149.112.112.112/dns-query", "Quad9-Recommended 149.112.112.112")
+	o:value("https://dns.quad9.net/dns-query", "Quad9-Recommended")
```

Comment:
why do you want to use domain instead of ip?
and you don't specify the ip (like line 336 and line 337)